plugin/xcodebuild.vim

```diff
@@ -0,0 +1,49 @@
+command! XcodebuildBuild call <sid>build()
+command! XcodebuildTest call <sid>test()
+
+let s:plugin_path = expand("<sfile>:p:h:h")
+
+function! s:build()
+  let cmd = s:base_command()  . s:xcpretty()
+  call s:run_command(cmd)
+endfunction
+
+function! s:test()
+  let cmd =  s:base_command() . ' -sdk iphonesimulator test' . s:xcpretty()
+  call s:run_command(cmd)
+endfunction
+
+function! s:run_command(cmd)
+  echom a:cmd
```

Comment:
Hmm, is this valuable? Does the next `execute` line drop people into a shell anyway, thus wiping the messages?